client/src/pages/Register/PayPalButtons.tsx

```diff
@@ -50,30 +70,8 @@ function PayPalButtonsComponent(props: Props) {
   }) as MutableRefObject<InstanceData>;
   instance.current.props = props;
 
-  const optionsFromConfig = getFromPath(props, 'config.payPalOptions');
-
-  debug('formData', props.formData);
-  debug('config', props.config);
-
-  if (!optionsFromConfig) return null;
-
-  const payPalOptions: PayPalScriptOptions = {
-    ...optionsFromConfig,
-    // put overrides here
-    // ....
-  };
-
-  if (process.env.NODE_ENV === 'development' && !payPalOptions['client-id']) {
-    payPalOptions['client-id'] = 'sb';
-  }
-
   return (
-    <PayPalProvider
```

Comment:
This has been moved to the RegisterStep component, since this is the only way to get the listener/script destroyer to work